configs/sample_yamls/meshops/handlers.yml

```diff
@@ -0,0 +1,21 @@
+handlers:
+- name: versionList
+  adapter: genericListChecker
```

Comment:
Given generalListChecker_params below, do we need this adapter field? Seems redundant.

@mandarjog @ZackButcher @douglas-reid